src/components/profile/PhoneInput.css

```diff
@@ -8,6 +8,10 @@
   flex-grow: 1;
 }
 
+.edit_profile_phone_input .react-phone-number-input--invalid {
```

Comment:
1. could we go RN-way here

a. define invalid style in Stylesheet with border-color: #fa6c77;
b. add invalid style to styles when input state is invalid 

?

2.how and where `react-phone-number-input--invalid` css class  it's adding ? why it has `--` suffix instead of `__` like in the next one `react-phone-number-input__input` ?
It's something you've added or somethind from phone number component implementation ?
Let's align those suffixes. Let both they will be `__` (BEM-like)

Reply:
@serdiukov-o-nordwhale we can't control class names of phone input as it comes from the phone-number-input library. Also, we cant define error styles for input via StyleSheets as there is no way to apply it to the correct element. So CSS file used to override phone lib styles.